api.raml

```diff
@@ -0,0 +1,107 @@
+#%RAML 1.0
+title: ID Broker API
+protocols: [ HTTPS ]
+mediaType: application/json
+securitySchemes:
+  # TODO: Define auth. bearer token schema
+securedBy:
+  # TODO: Specify auth. bearer token schema
+types:
+  Authentication:
+    type: object
+    properties:
+      username: string
+      password: string
+  User:
+    type: object
+    properties:
+      employee_id: string
+      first_name: string
+      last_name: string
+      display_name?: string
+      username: string
+      email: string
+      active?: YesOrNo
+      locked?: YesOrNo
+  YesOrNo:
+    type: string
+    enum: [ yes, no ]
+/authentication:
+  post:
+    body:
+      application/json:
+        type: Authentication
+        example: |
+          {
+            "username": "john_smith",
+            "password": "correcthorsebatterystaple"
+          }
+    responses:
+      200:
+        description:
+          The given credentials were acceptable/valid (i.e. - the
+          authentication succeeded).
+        body:
+          application/json:
+            description: The attributes of the authenticated User.
+            type: User
+            example: |
+              {
+                "employee_id": "12345",
+                "first_name": "John",
+                "last_name": "Smith",
+                "display_name": "John Smith",
+                "username": "john_smith",
+                "email": "john_smith@example.com",
+                "active": "yes",
+                "locked": "no"
```

Comment:
@fillup Do we want these timestamps coming out? It doesn't seem like we would need (want?) them on an authentication call, but I don't know what other situations you might have had in mind.